src/tests/reflection/GenericAttribute/GenericAttributeMetadata.il

```diff
@@ -13,14 +13,14 @@
 
   // --- The following custom attribute is added automatically, do not uncomment -------
   //  .custom instance void [mscorlib]System.Diagnostics.DebuggableAttribute::.ctor(valuetype [mscorlib]System.Diagnostics.DebuggableAttribute/DebuggingModes) = ( 01 00 07 01 00 00 00 00 ) 
-
+/*  Re-enable once the fix to dotnet/msbuild#6734 propagates to this repo.
```

Comment:
Instead of this format, I would use the full github URL. it is easier to grep for.
```suggestion
/*  Re-enable once the fix to https://github.com/dotnet/msbuild/issues/6734 propagates to this repo.
```